Test/allocated1/dafny0/Twostate-Functions.dfy.expect

```diff
@@ -6,17 +6,17 @@ Twostate-Functions.dfy(57,14): Related location
 Twostate-Functions.dfy(72,15): Error: assertion might not hold
 Twostate-Functions.dfy(57,14): Related location
 Twostate-Functions.dfy(96,24): Error: argument ('u') might not be allocated in the two-state function's previous state
-Twostate-Functions.dfy(101,40): Error: argument 1 ('x') might not be allocated in the two-state function's previous state
-Twostate-Functions.dfy(107,40): Error: argument 1 ('x') might not be allocated in the two-state function's previous state
+Twostate-Functions.dfy(101,40): Error: argument at index 1 ('x') might not be allocated in the two-state function's previous state
```

Comment:
Yay thanks in the name of all old-fashioned thinkers like me who still believe that argument 1 is the first argument and not the second :-)